app/services/done_index_job.rb

```diff
@@ -63,8 +68,14 @@ def _call
     if successful?
       ran_job.cleanup_when_done
     else
-      Raven.capture_message("Job in Error Found in Done Queue", :extra => inspect)
-      ran_job.remove_associated_book_index_state
+      if http_error?
+        Raven.capture_message("Openstax http error", :extra => inspect)
```

Comment:
Can remove "Openstax".

What does a second Raven capture get for us here? (with the other one in the worker)